Makefile

```diff
@@ -43,7 +44,10 @@ clean:
 	rm -rf coverage.txt
 
 .PHONY: thrift
-thrift:	thrift-image clean $(THRIFT_FILES)
+thrift: thrift-image clean
```

Comment:
see proto targets that distinguish between all-languages generation and committed Go-only generation